engine/chain/internal/bhs/verify_mr.go

```diff
@@ -0,0 +1,76 @@
+package bhs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+
+	"github.com/bitcoin-sv/go-paymail/spv"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+)
+
+/**
+NOTE: When you switch from enabled use_beef to disabled and restart the server,
+sometimes paymail capabilities are cached and VerifyMerkleRoots is called even though it shouldn't be.
+*/
```

Comment:
Do we need this note 🙈 
this service could be called in any other flow, not only paymail, and nobody will care to look if there is some comment with note.

Reply:
We don't need it - but when I delete it, this knowledge will stay only in our heads :D 

But ok, removed 👍 I agree that this is not the best for it.